### PR TITLE
add subscription themes consent to app native logs

### PR DESCRIPTION
**Pull request by MatthieuRipollPassCulture**
# DA PR

## Describe your changes

Please include a summary of the changes: Add subscription info (current state, new subscriptions) to app native logs

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @cdelabre @carolinemestre-passculture 

## Jira ticket number and/or notion link

JIRA-ticket_number https://passculture.atlassian.net/browse/PC-29468

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [x] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Fields have been snake_cased
- [ ] I have checked my modifications don't break downstream models
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I have updated the dag in cases of dependencies
- [ ] My code passes CI/CD tests
- [x] I will post on slack review channel and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### CODE REVIEW

File: orchestration/dags/dependencies/logs/sql/analytics/app_native_logs.sql [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+    jsonPayload.extra.newlysubscribedto.themes AS newly_subscribed_themes,
+    jsonPayload.extra.newlysubscribedto.email AS newly_subscribed_email,
+    jsonPayload.extra.newlysubscribedto.push AS newly_subscribed_push,
+    jsonPayload.extra.subscriptions.subscribed_themes AS currently_subscribed_themes,
+    jsonPayload.extra.subscriptions.marketing_push AS currently_subscribed_marketing_push,
+    jsonPayload.extra.subscriptions.marketing_email AS currently_subscribed_marketing_email,
+    jsonPayload.extra.newlyunsubscribedfrom.themes AS newly_unsubscribed_themes,
+    jsonPayload.extra.newlyunsubscribedfrom.email AS newly_unsubscribed_email,
+    jsonPayload.extra.newlyunsubscribedfrom.push AS newly_unsubscribed_push,
+    FROM
```
Comment (carolinemestre-passculture) [inline orchestration/dags/dependencies/logs/sql/analytics/app_native_logs.sql:21]: @MatthieuRipollPassCulture Est-ce que tu penses que c'est possible de faire ça directement dans DBT pour limiter le legacy? On a le modèle `int_pcapi__log` dans lequel tu peux rajouter les champs manquants si nécessaire, puis ensuite le modèle `int_pcapi__native_log` dans lequel tu pourrais appeler les nouveaux champs ?

Comment (MatthieuRipollPassCulture) [reply orchestration/dags/dependencies/logs/sql/analytics/app_native_logs.sql:21]: J'ai suivi la reco de @cdelabre qui voulait éviter un double run et scinder les donnée pcapi (calls HTTP) des logs sur l'app native, mais à dispo pour en discuter tous les 3 !

Comment (carolinemestre-passculture) [reply orchestration/dags/dependencies/logs/sql/analytics/app_native_logs.sql:21]: Ok ce sont deux modèles qui sont pour la partie native, se basent sur `raw.stdout` mais sont distinct. On pourrait quand même faire deux modèles dans intermediate, et yes on peut en rediscuter ensemble :)

Comment (MatthieuRipollPassCulture) [reply orchestration/dags/dependencies/logs/sql/analytics/app_native_logs.sql:21]: Qu'est ce qu'on fait du coup @carolinemestre-passculture ?

**Pull request merged**